labs/06_scale.md

```diff
@@ -62,7 +65,7 @@ Zum Schluss schauen wir uns den Service an. Der sollte jetzt alle drei Endpoints
 ```bash
 $ oc describe svc appuio-php-docker
 Name:              appuio-php-docker
-Namespace:		techlab-scale
+Namespace:              techlab-scale
```

Comment:
This one does not make sense.

Reply:
Good catch, @bliemli! I guess there were some mixed spaces and tabs here ;-) Should be fixed now.